build.gradle

```diff
@@ -211,3 +168,8 @@ bootJar {
     attributes('Implementation-Version': project.version.toString())
   }
 }
+
+// Gradle 7.x issue, workaround from: https://github.com/gradle/gradle/issues/17236#issuecomment-894768083
+rootProject.tasks.named("processIntegrationTestResources") {
```

Comment:
I've never seen this needed before on other projects that have multiple test sets. The existing build setup seems nonstandard, i'll try and repro myself.

Reply:
it's in the spring boot template and quite a few projects have it, probably misconfiguration